test/service/test_manifest.py

```diff
@@ -975,6 +997,55 @@ def test_full_metadata_missing_fields(self):
             self.assertGreater(len(intersection), 0)
             self.assertGreater(len(symmetric_diff), 0)
 
+    def test_manifest_format_validation(self):
+        url = self.base_url + '/manifest/files?format=invalid-type'
+        response = requests.get(url)
+        self.assertEqual(400, response.status_code, response.content)
+
+    @mock_sts
+    @mock_s3
+    def test_manifest_content_disposition_header(self):
+        self._index_canned_bundle(("f79257a7-dfc6-46d6-ae00-ba4b25313c10", "2018-09-14T133314.453337Z"))
+        filtered_uuids = {True: '4ad2e5aa-dcbd-5fed-ac3e-17bcd82c8c0f',
+                          False: '525cebc4-b96a-5f75-b329-8268fe947788'}
+        unfiltered_uuids = {True: '4a5cd62a-8c55-5ddf-94e5-86c51f7603cd',
+                            False: '585ac408-338e-5e3e-a856-e4df962509b6'}
+        with mock.patch.object(manifest_service, 'datetime') as mock_response:
+            mock_response.now.return_value = datetime(1985, 10, 25, 1, 21)
+            storage_service = StorageService()
+            storage_service.create_bucket()
+            for single_part in True, False:
+                for filters, expected_name in [
+                    # For a single project, the content disposition file name should
+                    # be the project name followed by the date and time
+                    (
+                        {'project': {'is': ['Single of human pancreas']}},
+                        'Single of human pancreas 1985-10-25 01.21'
+                    ),
+                    # In all other cases, the standard content disposition file name
+                    # should be "hca-manifest-" followed by the manifest key, a
+                    # v5 UUID deterministically derived from the filter and
+                    (
+                        {'project': {'is': ['Single of human pancreas', 'Mouse Melanoma']}},
+                        f'hca-manifest-{filtered_uuids[single_part]}',
```

Comment:
replace the dict lookup with a conditional. To prevent and overly long line it may help to convert the f string into concatenation